docs/news.rst

```diff
@@ -3,6 +3,158 @@
 Release notes
 =============
 
+1.1.0
+-----
+
+This 1.1 release brings a lot of interesting features and bug fixes:
+
+- Scrapy 1.1 has basic Python 3 support (requires Twisted >= 15.5). See
+  :ref:`news_basicpy3` for more details and some limitations.
+- Hot new features:
+
+  - ItemLoaders now support nested loaders (:issue:`1467`).
+  - ``FormRequest.from_response`` improvements (:issue:`1382`, :issue:`1137`).
+  - Added setting :setting:`AUTOTHROTTLE_TARGET_CONCURRENCY` and improved
+    AutoThrottle docs (:issue:`1324`).
+  - Added ``response.text`` to get body as unicode (:issue:`1730`).
+  - Anonymous S3 connections (:issue:`1358`).
+  - Deferreds in downloader middlewares (:issue:`1473`). This enables better
+    robots.txt handling (:issue:`1471`).
+  - HTTP cache improvements (:issue:`1151`).
+
+- These bug fixes may require your attention:
+
+  - Don't retry bad requests (HTTP 400) (:issue:`1289`).
+  - Fix shell files argument handling (:issue:`1710`, :issue:`1550`).
+  - Fixes on robots.txt handling (:issue:`1783`).
+  - Exporters work on unicode (:issue:`1080`).
+  - Fix xmliter to accept nodenames with dots (:issue:`1533`).
+
+Keep reading for more details on other improvements and bug fixes.
+
+.. _news_basicpy3:
+
+Basic Python 3 Support
+~~~~~~~~~~~~~~~~~~~~~~
+
+We have been hard at work to make Scrapy run on Python 3. As a result, now you
+can run spiders on Python 3.3, 3.4 and 3.5 (Twisted >= 15.5 required). Some
+features are still missing (and some may never be ported).
+
+Almost all addons/middlewares are expected to work. However, we are aware of
+some limitations:
+
+- S3 downloads are not supported (see :issue:`1718`)
+- Sending emails is not supported
+- FTP download handler is not supported (non-Python 3 ported Twisted
+  dependency)
+- Telnet is not supported (non-Python 3 ported Twisted dependency)
+- There are problems with non-ASCII URLs in Python 3
+- Reported problems with HTTP caches created by Scrapy in Python 2.x which
+  can't be reused in Scrapy in Python 3.x (to be checked)
+- There is also a nasty issue with `cryptography` library: recent versions
```

Comment:
Maybe we should mention this is not a Scrapy bug, but a problem users should be aware of if they run Scrapy in Python 3.5. Also, as it is not clear if the issue is with cryptography or cffi, we shouldn't blame them :)